src/service/rankingService.js

```diff
@@ -0,0 +1,99 @@
+let user =  {
+  name: 'Player1',
+  score: 15,
+  timeInSeconds: 93
+};
+// check localStorage for current Top scores
+// if empty create object with objects mode1, mode2, and mode3
+// PokemonApiRanking - data from/to localStorage
+const PokemonApiRanking = JSON.parse(localStorage.getItem('PokemonApiRanking')) || {
```

Comment:
You should get it every time you call `rankingService` function, not only at the start.

If you once add score to localStorage then this PokemonApiRanking is updated, okay... but the second score that would be added is based not on the content of the local storage (that could be somehow changed between first and second game), but is based on the local copy you have in this variable. (Not sure if this is clear enough - if not - contact me)